serde/src/test/org/apache/hadoop/hive/serde2/avro/TestAvroSerializer.java

```diff
@@ -125,7 +125,7 @@ public void canSerializeDoubles() throws SerDeException, IOException {
   public void canSerializeTimestamps() throws SerDeException, IOException {
     singleFieldTest("timestamp1", Timestamp.valueOf("2011-01-01 00:00:00").toEpochMilli(),
         "\"" + AvroSerDe.AVRO_LONG_TYPE_NAME + "\"," +
-        "\"logicalType\":\"" + AvroSerDe.TIMESTAMP_TYPE_NAME + "\"");
+        "\"logicalType\":\"" + AvroSerDe.TIMESTAMP_TYPE_NAME_MILLIS + "\"");
```

Comment:
Its better to extend test classes one for each of these datatypes, rather than replace this by TIMESTAMP_TYPE_NAME_MILLIS.

Reply:
We needed a change here as our avro schemas in hive by default are now serialized to have microsecond precision.

Reply:
The only change made here is nomenclature, basically TIMESTAMP_TYPE_NAME before used to mean TIMESTAMP_TYPE_NAME_MILLIS by default, but now that we have added the an extra logical type TIMESTAMP_TYPE_NAME_MICROS, the differentiation needed to be made between TIMESTAMP_TYPE_NAME and TIMESTAMP_TYPE_NAME_MICROS.
Hence, the test itself is not changed, TIMESTAMP_TYPE_NAME is now renamed to TIMESTAMP_TYPE_NAME_MILLIS of the Avroserde class

Explicit tests for micros are added separately